superset-frontend/package.json

```diff
@@ -266,6 +266,7 @@
     "less": "^3.9.0",
     "less-loader": "^5.0.0",
     "mini-css-extract-plugin": "^0.4.0",
+    "node-fetch": "^2.6.0",
```

Comment:
this is an internal dependency for `fetchMock`. Installing this was necessary in order to mock `blob` responses

Reply:
[optional] this would be an excellent comment to squeeze into the `package.json` if json supported comments. There are some hacky ways to add comments. https://coding.napolux.com/how-to-add-comments-to-package-json/

Reply:
I'm not in favor of hacking in json comments. This is installed under devDependencies and the git blame should be enough to trace the change back to this PR.

Reply:
`package.json5` -- one can dream, right? ;)